code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm

```diff
@@ -238,6 +238,18 @@
 		if(dose == removed)
 			S.visible_message("<span class='warning'>[S]'s flesh sizzles where the water touches it!</span>", "<span class='danger'>Your flesh burns in the water!</span>")
 
+
+/datum/reagent/water/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	if(istype(M, /mob/living/carbon/slime))
+		var/mob/living/carbon/slime/S = M
+		S.adjustToxLoss(20 * removed) // A slime having water forced down its throat would cause much more damage then being splashed on it
+		if(!S.client)
+			if(S.Target) // Like cats
+				S.Target = null
+				++S.Discipline
+		if(dose == removed)
+			S.visible_message("<span class='warning'>[S]'s screams in pain as you force water down its throat!</span>", "<span class='danger'>Your insides burn!</span>")
```

Comment:
A visible message is also not needed here, I don't think that slimes even feel people or scream when in contact with water, at most, they just fizzle when it touches them.

Reply:
This message cannot be used anyways. Since we cannot ensure that the slime is being forcefed from this proc. So, no visible message here.

Reply:
Grammar is invalid too.